lib/pxenforcer.js

```diff
@@ -64,32 +65,37 @@ class PxEnforcer {
     }
 
     enforce(req, res, cb) {
-        const requestUrl = req.originalUrl;
+        if (!this._config.ENABLE_MODULE) {
+            this.logger.debug('Request will not be verified, module is disabled');
+            return cb();
+        }
+        const normalizedUrl = this._normalizeUrl(pxUtil.getFullUrlFromRequest(req));
+        const NormalizedWholeUrl = normalizedUrl.href;
+        req[requestUrlKeys.requestNormalizedUrl] = NormalizedWholeUrl;
+        req[requestUrlKeys.isOriginalDifferentFromOriginal] = req.originalUrl !== NormalizedWholeUrl;
+        
+        const NormalizedPath = normalizedUrl.pathname;
```

Comment:
1. Variables should start with lowercase letter. Convention is `variableStyle`, `CONSTANT_STYLE`, and `ClassStyle`.
2. I'd remove the use of `requestUrlKeys` - it only needs to be used in 2 places (here and in PXContext), and it's not used in PXContext anyway.
3. I think you can just use `normalizedUrl.href` instead of creating a new variable.